src/snapred/backend/dao/StateId.py

```diff
@@ -6,14 +6,14 @@
 class StateId:
     vdet_arc1: float
     vdet_arc2: float
-    WavelengthUserReq: float
+    WavelengthReq: float
```

Comment:
try
``` python
WavelengthReq: float = Field(..., serialization_alias="WavelengthUserReq")
```
That should return the stateId values back to their originals.  If not, you will need to re-name this field to `WavelgnthUserReq`

Reply:
Yes, this makes a lot more sense.

Reply:
You'll need to change the state Ids in the tests back